src/main/js/html.js

```diff
@@ -663,6 +664,7 @@
 
         if (first.tagName === "SPAN" &&
             second.tagName === "SPAN" &&
+            first._isd_element &&
```

Comment:
This line will mask the impact of upstream code not setting `_isd_element` when we want that to happen, so possibly better not to add it?

Reply:
Hmm, I'm not sure in what circumstances _isd_element would be unset, but almost certainly we'd not want to combine spans which had not come from the same source element.  I think it adds robustness.  If we just want to highlight errors as well as being robust, we could move the test inside the if, and console.error? no access to context.errorHandler there at the moment :(.

Reply:
That's a good point @btsimonh , I hadn't thought of it like that.